protocol/contracts/TrucoMatch.sol

```diff
@@ -64,13 +64,14 @@ contract TrucoMatch {
         IERC3333 _trucoEngine,
         IERC20 _truCoin,
         GameStateQueries _gameStateQueries,
+        address player1,
         uint256 _bet
     ) {
         trucoEngine = _trucoEngine;
         truCoin = _truCoin;
         gameStateQueries = _gameStateQueries;
         currentMatch.bet = _bet;
-        currentMatch.players[0] = msg.sender;
```

Comment:
It remained from when we hadn't Factory contract, assuming that the person that starts the match is the player1.
It could be **tx.origin**, cause the action is originated by player1.